src/components/ui/form.tsx

```diff
@@ -144,9 +144,12 @@ function FormDescription({
       data-slot="form-description"
       id={formDescriptionId}
       className={cn('text-base-color text-xs', className)}
+      {...(typeof children === 'string'
```

Comment:
We should prevent XSS exposure and therefore we should not accept whatever comes in the children. We can use [DomPurify](https://github.com/cure53/DOMPurify) like we do at Remote.